samples/nextjs/src/lib/page-props-factory.ts

```diff
@@ -131,7 +137,13 @@ export class SitecorePagePropsFactory {
           componentModule,
         });
       }
+
+      this.layoutPersonalizationUtils.replacePersonalizableComponentsWithLoaderComponents(
+        layoutData.sitecore.route?.placeholders,
+        'PersonalizationLoadingComponent'
```

Comment:
Hm, so, in case if we changed this name in Sitecore, we need to go here and change this name manually? Can it be somehow tied to this component? Instead of having separate constant

Reply:
This name is not something defined by Sitecore. It is just the name of the loading component in the frontend application.
samples/nextjs/src/components/**PersonalizationLoadingComponent**.tsx